tasks/update_go.py

```diff
@@ -138,95 +147,38 @@ def _get_repo_go_version() -> str:
     return version.strip()
 
 
-# extracts the major version from the given string
+# extracts the minor version from the given string
 # eg. if the string is "1.2.3", returns "1.2"
-def _get_major_version(version: str) -> str:
+def _get_minor_version(version: str) -> str:
     import semver
```

Comment:
Do we know why this `import` isn't at the top of the file? I noticed it appears in `update_go` as well.

Reply:
Some of the imports are not located at the top of the file because we do not want to import it all the time.
Here for example we import `semver` only if we use `_get_minor_version` function. With that we do not need to have `semver` installed in all the images, only in the one that actually call the `_get_minor_version`

Reply:
That's exactly why, note that I'm actually moving `semver` directly to the buildimages so that I can use it in the tests of the invoke task, so I'll move that import back to the top when it's done